cdk/lib/security-hq.ts

```diff
@@ -86,6 +86,17 @@ export class SecurityHQ extends GuStack {
 
     const domainName = "security-hq.gutools.co.uk";
 
+    const userData = UserData.forLinux();
+    userData.addCommands(`#!/bin/bash -ev
```

Comment:
Looking at the snapshot, it looks like CDK is adding the shebang line for us and we don't need to add `#!/bin/bash -ev` ourselves.

Reply:
ah yes! thanks Akash, I forgot to remove that bit.